server/src/main/java/org/opensearch/action/bulk/TransportBulkAction.java

```diff
@@ -756,6 +762,44 @@ public void onTimeout(TimeValue timeout) {
             });
         }
 
+        private boolean addFailureIfIndexAppendOnlyAndOpsDeleteOrUpdate(
```

Comment:
Can you add UTs for this in `TransportBulkActionTests`?

Reply:
Code coverage is already taken care by integ tests.